titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/ext/BatchJobExt.java

```diff
@@ -34,7 +34,7 @@
     @FieldInvariant(value = "value <= @constraints.getMaxBatchJobSize()", message = "Batch job too big #{value} > #{@constraints.getMaxBatchJobSize()}")
     private final int size;
 
-    @Min(value = 60_000, message = "Runtime limit too low (must be at least 60sec, but is #{#root}[ms])")
+    @Min(value = 5_000, message = "Runtime limit too low (must be at least 5sec, but is #{#root}[ms])")
```

Comment:
Good catch. I was planning to update this as well and forgot it. I will get it fixed.

Reply:
@tbak since we already check that `state == TaskState.Started` here, I don't think we need to account for propagation delays. I made the fix to always use the same `min` from `BatchJobExt`, but that is just a safeguard.

Let me know what you think.